docs/guides/tutorials/getting_started/customize_your_deployment.rst

```diff
@@ -1,9 +1,9 @@
 .. _tutorials__getting_started__customize_your_deployment:
 
-Customize your deployment
+Optional: Customize your deployment
```

Comment:
I thought we were going to pull this out into its own top-level tutorial.

Is this an intermediate step towards that end?

Reply:
See my comment in the initial commit message - the section is very much not written as a tutorial and really just contains a few links, so it just made more sense to keep it as an optional "where to go from here".

Reply:
Oh and regarding this being an intermediate step I'd say we could think of making this a full-on tutorial, but I didn't realize how far it was from being self-standing as a tutorial, so not for now.

Reply:
🤦 for not reading your original commit message.

Yes, this makes sense.